### PR TITLE
Simplify gravity computation (removing trigonometry)

### PR DESCRIPTION
**Pull request by DroneBetter**
Compute gravity axially, make acceleration iterate pairwise without duplicates for half the distance coefficient computations

### CODE REVIEW

**Review by CommandCracker8 — CHANGES_REQUESTED** [review]
It doesn't work. Errors when I run it

File: PhysicsSimulation.py [hunk 1]
```diff
@@ -265,20 +236,19 @@ def to_pygame(coords):
...
-                    movementAmount = ((Fgravity / object1.mass) * system.runtimeScale) / fps
-
-                    acceleration, dist_coords, theta, position, flip = calculate_acceleration_coordinates(object1, object2, movementAmount)
-                    log("Distance Coords: ", dist_coords)
-                    print(flip)
-                    log("Theta: ", theta * 57.2958)
-                    log("Acceleration: ", object1.acceleration)
-
-                    object1.acceleration = acceleration
+        for i,object1 in enumerate(objects):
```
Comment (CommandCracker8) [inline PhysicsSimulation.py:239]: system.

**Comment by CommandCracker8** [comment]
Line 243 awaits your attention.

**Comment by CommandCracker8** [comment]
37 and 38
`NameError: name 'normalised_dist_coords' is not defined. Did you mean: 'normalized_dist_coords'?`

**Comment by DroneBetter** [comment]
Darn Americans

**Comment by CommandCracker8** [comment]
The direction the moon travels in is now reversed.

**Comment by CommandCracker8** [comment]
:D

**Pull request merged**